mirrorlist_test.go

```diff
@@ -88,7 +83,7 @@ repos:
 		PurgeFilesAfter: 2592000,
 		DownloadTimeout: 200,
 	}
-	if !cmp.Equal(*got, *want) {
+	if !cmp.Equal(*got, *want, cmpopts.IgnoreUnexported(Repo{})) {
```

Comment:
same here

Reply:
I've changed these to only ignore the mutex. Let me know if that's all right